ci/test-miri.sh

```diff
@@ -10,6 +10,9 @@ _ cargo "+${rust_nightly}" miri test -p solana-program -- hash:: account_info::
 
 _ cargo "+${rust_nightly}" miri test -p solana-unified-scheduler-logic
 
+# test big endian branch
```

Comment:
No, why do you ask? Our tests all run on little endian targets normally so I just added this specifically for testing on a big endian target

Reply:
We run on little endian targets but we don't run on _miri_ for little endian targets. We have more unsafe code for little endian than big endian currently, so we should run it for little endian too to catch UB.

Reply:
Ah I see, I think we weren't understanding each other because I'm only using miri here because it's useful for targeting big endian (nothing about UB). But yeah happy to add it for little endian to potentially catch UB in our unsafe code too.